source/slang/slang-ir-legalize-types.cpp

```diff
@@ -3427,21 +3228,6 @@ static LegalVal declareVars(
 {
     LegalVarChain varChain = inVarChain;
     IRTypeLayout* typeLayout = inTypeLayout;
-    if (isSpecial)
```

Comment:
This is another case where I'd like to better understand whether `isSpecial` should be considered an error case or not, during this legalization pass.

Reply:
tracked in https://github.com/shader-slang/slang/issues/8873